packages/theme/src/skeleton.ts

```diff
@@ -9,13 +11,31 @@ export const standardSkeletonColors = {
     skeletonForegroundElevation1: '#f6f6f6',
 };
 
+type SkeletonColor = keyof typeof standardSkeletonColors;
+
 export const darkSkeletonColors = {
-    skeletonBackgroundElevationNegative: '#eeeeeeff',
-    skeletonForegroundElevationNegative: '#f6f6f6ff',
+    skeletonBackgroundElevationNegative: '#000000ff',
+    skeletonForegroundElevationNegative: '#161716ff',
 
-    skeletonBackgroundElevation0: '#f6f6f6',
-    skeletonForegroundElevation0: '#eeeeee',
+    skeletonBackgroundElevation0: '#161716ff',
+    skeletonForegroundElevation0: '#000000ff',
 
-    skeletonBackgroundElevation1: '#ffffff',
-    skeletonForegroundElevation1: '#f6f6f6',
+    skeletonBackgroundElevation1: '#ffffffff',
+    skeletonForegroundElevation1: '#161716ff',
+};
+
+export const mapElevationToSkeletonBackground: Record<Elevation, SkeletonColor> = {
+    '-1': 'skeletonBackgroundElevationNegative', // For example left menu is negative elevation
+    0: 'skeletonBackgroundElevation0',
+    1: 'skeletonBackgroundElevation1',
+    2: 'skeletonBackgroundElevation0',
+    3: 'skeletonBackgroundElevation1',
+};
```

Comment:
As per my Slack message, the original Figma frame was misleading. Skeletons do not have a background but they have their own elevation mapping, so having 1 object should be enough :)

Reply:
:heavy_check_mark: https://github.com/trezor/trezor-suite/pull/10731/commits/efe03630c674f71d931529940315c511a5ee1fa2

Reply:
nit: maybe drop the `foreground` naming as well?